docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/vimeo-tracking/index.md

```diff
@@ -0,0 +1,207 @@
+---
+title: 'Vimeo Tracking'
+date: '2022-01-12'
+sidebar_position: 17000
+---
+
+```mdx-code-block
+import Block5966 from '@site/docs/reusable/javascript-tracker-release-badge-v3/_index.md'
+<Block5966/>
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+This plugin enables the automatic tracking of a Vimeo video, utilising the [Snowplow Media Plugin.](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/plugins/media)
+
+## Installation
+
+- `npm install @snowplow/browser-plugin-vimeo-tracking`
+- `yarn add @snowplow/browser-plugin-vimeo-tracking`
+- `pnpm add @snowplow/browser-plugin-vimeo-tracking`
+
+## Initialization
+
+```javascript
+import { newTracker, trackPageView } from '@snowplow/browser-tracker';
+import { VimeoTrackingPlugin, startVimeoTracking } from '@snowplow/browser-plugin-vimeo-tracking';
+
+newTracker('sp1', '{{collector_url}}', { 
+   appId: 'my-app-id', 
+   plugins: [ VimeoTrackingPlugin() ],
+});
+
+startVimeoTracking(/* options);
+```
+
+:::info Example app
+To illustrate the tracked events and entities, you can visit an example app that showcases the tracked media events and entities live as you watch a video.
```

Comment:
Make “events” and “entities” links to the respective pages, as per the repo README (see the section on concepts)